src/js/views/login/translations/pt_br.login.i18n.json

```diff
@@ -6,5 +6,6 @@
   "User is required": "Usuário é Obrigatório",
   "5 characters minimum": "Mínimo de 5 caracteres",
   "Type the password": "Digite a senha",
-  "Password is required": "Senha é obrigatório"
-}
+  "Password is required": "Senha é obrigatório",
+  "Error": "Login ou senha inválidos"
```

Comment:
If was not possible communication with back? What's error will be showing? 
If the error is "Login ou senha inválidos" maybe Its better change for something like "Erro ao ao tentar se logar".

Reply:
I changed the login error message and added the network error